timemachine/md/minimizer.py

```diff
@@ -26,6 +28,15 @@ class MinimizationError(Exception):
     pass
 
 
+def check_force_norm(du_dx):
+    """raise Minimization error if the force on any atom exceeds MAX_FORCE_NORM"""
+    norm = np.linalg.norm(du_dx, axis=-1)
+    if not np.all(norm < MAX_FORCE_NORM):
```

Comment:
nit: seems a bit odd to me that this refers to a global constant (vs. optional argument). I see that the constant is used in one other place (https://github.com/proteneer/timemachine/blob/195ce1c0f9cebc7f184b5322dac2faba0345ef7e/timemachine/fe/model_utils.py#L43), but it's a bit unclear to me that it makes sense to reuse the same value in both contexts.

Reply:
Intent was just to reduce repetition of this check within `minimizer.py` -- the suggestion of an optional argument makes sense, applied in https://github.com/proteneer/timemachine/pull/978/commits/743706d4972fe3adddc1c4a62b1950986b97589b